packages/tables/resources/views/components/header.blade.php

```diff
@@ -1,52 +1,66 @@
 @php
     use Filament\Support\Enums\Alignment;
+    use Filament\Support\Enums\IconSize;
     use Filament\Tables\Actions\HeaderActionsPosition;
 @endphp
 
 @props([
     'actions' => [],
     'actionsPosition',
-    'description' => null,
+    'icon' => null,
+    'iconColor' => 'gray',
+    'iconSize' => IconSize::Large,
     'heading' => null,
+    'description' => null,
```

Comment:
Keep this sorted alphabetically please :)

Reply:
thanks @zepfietje I will follow your instructions